TestResultSummaryService/routes/getFeedbackUrl.js

```diff
@@ -0,0 +1,14 @@
+module.exports = async ( req, res ) => {
+    const { repoName, buildName, issueName, issueCreator, accuracy } = req.query;
+
+    let feedbackUrlRes = {};
+
+    feedbackUrlRes.repoName = repoName;
+    feedbackUrlRes.buildName = buildName;
+    feedbackUrlRes.issueName = issueName;
+    feedbackUrlRes.issueCreator = issueCreator;
+    feedbackUrlRes.accuracy = true;
```

Comment:
To test if the backend receives the info, you can print it to check when developing.